G/GEOS/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "GEOS"
+version = v"3.8.0"
+
+# Collection of sources required to build GEOS
+sources = [
+    "http://download.osgeo.org/geos/geos-$src_version.tar.bz2" =>
```

Comment:
I guess you meant `$(version)` instead of `$src_version`?

Reply:
Good catch, fixed it now.